cv_bridge/src/module.hpp

```diff
@@ -37,6 +37,7 @@ PyObject* pyopencv_from(const cv::Mat& m);
 static int do_numpy_import( )
```

Comment:
Should we update the signature to `void *`? This is what I'm proposing in the equivalent `ros2` patch (https://github.com/ros-perception/vision_opencv/pull/346).